src/promptflow-devkit/tests/sdk_cli_test/e2etests/test_flow_test.py

```diff
@@ -431,3 +434,53 @@ def test_flex_flow_with_init(self, pf):
         result2 = pf.test(flow=flow_path, inputs={"func_input": "input"}, init={"obj_input": "val"})
         assert result2["func_input"] == "input"
         assert result1["obj_id"] != result2["obj_id"]
+
```

Comment:
Maybe we need also add tests in pf-core?

Reply:
Sure, added